source/css/sass/mixins/_typography.scss

```diff
@@ -17,48 +18,48 @@
 
 @mixin h1-typography() {
   @include _heading-base-typography();
-  @include set-font-size-and-line-height($font-size-h1, 48px);
+  @include set-font-size-and-line-height(ms(7), $baselinegrid-space-medium);
```

Comment:
Are these scale points explicitly documented? Seems hard to read / use atm.

Reply:
As it stands, we use: `-3`, `0`, `1`, `2`, `3`, `4`, `6`, `7`, `8`, `9` and `10`. Not sure we can give them sensible names.

Might be able to use intermediate variables which helps the readability here a little.

Reply:
It works well, but I think it needs something to make it feel more usable. Intermediate and / or Readme update would probably be worth it.

Reply:
I think changing the way config is handled can resolve this, ok to postpone?

Reply:
Sure. Can revisit this in the light of any config changes we introduce.